src/cur-mot/map-signatures-to-metadata.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+"""
+Map names in signature blocks to metadata.
```

Comment:
I'm not following how this is different from the other script that seemingly does the same